resources/simulations.ts

```diff
@@ -0,0 +1,81 @@
+import { BaseResource } from "."
+import { UnitConfig, UnitResponse } from "../types"
+import { AchReceivedPayment, Application, ApplicationDocument } from "../types"
+import {
+    ApproveApplicationSimulation,
+    DenyApplicationSimulation,
+    RejectDocumentSimulation,
+    CreateAchReceivedPaymentSimulation,
+} from "../types"
+
+export class Simulations extends BaseResource {
+    constructor(token: string, basePath: string, config?: UnitConfig) {
+        super(token, basePath + "/sandbox", config)
+    }
+
+    public async approveApplication(
+        applicationId: string,
+        request: ApproveApplicationSimulation
+    ): Promise<UnitResponse<Application>> {
+        return this.httpPost<UnitResponse<Application>>(
+            `/applications/${applicationId}/approve`,
+            {
+                data: request,
+            }
+        )
+    }
+
+    public async denyApplication(
+        applicationId: string,
+        request: DenyApplicationSimulation
+    ): Promise<UnitResponse<Application>> {
+        return this.httpPost<UnitResponse<Application>>(
+            `/applications/${applicationId}/deny`,
+            {
+                data: request,
+            }
+        )
+    }
+
+    public async approveDocument(
+        applicationId: string,
+        documentId: string
+    ): Promise<UnitResponse<ApplicationDocument>> {
+        return this.httpPost<UnitResponse<ApplicationDocument>>(
+            `/applications/${applicationId}/documents/${documentId}/approve`,
+            undefined
```

Comment:
good catch ... since the [approve document endpoint](https://docs.unit.co/simulations#approve-document) doesn't require any body params, I thought it might be OK to send empty body. I'm going to change this to empty document (`{}`) just to be safe.